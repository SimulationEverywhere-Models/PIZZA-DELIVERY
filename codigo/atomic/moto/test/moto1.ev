00:00:01:000 Carga_mi 3
00:00:02:000 Carga_mi 4
