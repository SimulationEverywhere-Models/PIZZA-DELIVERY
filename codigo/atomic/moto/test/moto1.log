Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para moto(02)
Mensaje D / 00:00:00:000 / moto(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / carga_mi /      3.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / carga_mi /      3.00000 para moto(02)
Mensaje D / 00:00:01:000 / moto(02) / 00:00:03:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / carga_mi /      4.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / carga_mi /      4.00000 para moto(02)
Mensaje D / 00:00:02:000 / moto(02) / 00:00:02:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:04:000 / Root(00) para top(01)
Mensaje * / 00:00:04:000 / top(01) para moto(02)
Mensaje Y / 00:00:04:000 / moto(02) / ocupada_mi /      1.00000 para top(01)
Mensaje D / 00:00:04:000 / moto(02) / 00:00:00:240 para top(01)
Mensaje Y / 00:00:04:000 / top(01) / ocupada_mi /      1.00000 para Root(00)
Mensaje D / 00:00:04:000 / top(01) / 00:00:00:240 para Root(00)
Mensaje * / 00:00:04:240 / Root(00) para top(01)
Mensaje * / 00:00:04:240 / top(01) para moto(02)
Mensaje Y / 00:00:04:240 / moto(02) / entrega_mi /      3.00000 para top(01)
Mensaje D / 00:00:04:240 / moto(02) / 00:00:00:060 para top(01)
Mensaje Y / 00:00:04:240 / top(01) / entrega_mi /      3.00000 para Root(00)
Mensaje D / 00:00:04:240 / top(01) / 00:00:00:060 para Root(00)
Mensaje * / 00:00:04:300 / Root(00) para top(01)
Mensaje * / 00:00:04:300 / top(01) para moto(02)
Mensaje Y / 00:00:04:300 / moto(02) / entrega_mi /      4.00000 para top(01)
Mensaje D / 00:00:04:300 / moto(02) / 00:00:00:300 para top(01)
Mensaje Y / 00:00:04:300 / top(01) / entrega_mi /      4.00000 para Root(00)
Mensaje D / 00:00:04:300 / top(01) / 00:00:00:300 para Root(00)
Mensaje * / 00:00:04:600 / Root(00) para top(01)
Mensaje * / 00:00:04:600 / top(01) para moto(02)
Mensaje Y / 00:00:04:600 / moto(02) / libre_mi /      1.00000 para top(01)
Mensaje D / 00:00:04:600 / moto(02) / ... para top(01)
Mensaje Y / 00:00:04:600 / top(01) / libre_mi /      1.00000 para Root(00)
Mensaje D / 00:00:04:600 / top(01) / ... para Root(00)
