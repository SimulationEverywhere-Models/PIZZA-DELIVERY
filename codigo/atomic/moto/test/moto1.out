00:00:04:000 ocupada_mi 1
00:00:04:240 entrega_mi 3
00:00:04:300 entrega_mi 4
00:00:04:600 libre_mi 1
